content/appointments.md

```diff
@@ -44,6 +44,6 @@ Think about your financial circumstances in general and plans for retirement, eg
 
 ##How to book
 
-Call 0300 330 1001 between 8am and 10pm, Monday to Sunday. Calls cost the same as a normal call - if your calls are free, it's included.
+Call 0300 330 1001 between 8am and 10pm, Monday to Sunday. Calls cost the same as a normal call - if your calls are free, it's included. Call 0203 733 3495 if you live abroad.
```

Comment:
Agreed

On 7 April 2015 at 12:24, Andrew Garner <notifications@github.com> wrote:

> In content/appointments.md
> <https://github.com/guidance-guarantee-programme/pension_guidance/pull/136#discussion_r27869271>
> :
>
> > @@ -44,6 +44,6 @@ Think about your financial circumstances in general and plans for retirement, eg
> >
> >  ##How to book
> >
> > -Call 0300 330 1001 between 8am and 10pm, Monday to Sunday. Calls cost the same as a normal call - if your calls are free, it's included.
> > +Call 0300 330 1001 between 8am and 10pm, Monday to Sunday. Calls cost the same as a normal call - if your calls are free, it's included. Call 0203 733 3495 if you live abroad.
>
> Should we say "you're calling from abroad" not "you live abroad" - then
> we're inclusive to people on business or holiday etc?
>
> We could qualify "abroad" as "outside the UK" since the main number is +44
> which is a UK number.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/guidance-guarantee-programme/pension_guidance/pull/136/files#r27869271>
> .
>